x-pack/plugins/lists/server/saved_objects/exception_list.ts

```diff
@@ -66,9 +66,18 @@ export const exceptionListMapping: SavedObjectsType['mappings'] = {
 
 export const exceptionListItemMapping: SavedObjectsType['mappings'] = {
   properties: {
-    comment: {
-      // TODO: investigate what the deep mapping structure of this really is
-      type: 'keyword',
+    comments: {
+      properties: {
+        comment: {
+          type: 'keyword',
```

Comment:
I'm curious if we could change comment type to `text` instead of `keyword` so the full content of the comment is searchable?

Reply:
Good call. I noticed other ones like `description` was set to `keyword`. Wasn't sure if that was for a specific reason so I defaulted to `keyword`. Will update!

Reply:
No worries I was just curious if it's something we might want. Open to hearing from others with their thoughts on this. Not a big deal either way I think.

Reply:
Per our chat, I'm planning on updating these in the upcoming PR that addresses `entries`. I'll link to it once its up as a follow up.